includes/reader-revenue/my-account/class-woocommerce-my-account.php

```diff
@@ -405,7 +405,7 @@ public static function is_user_verified() {
 	 */
 	public static function redirect_to_account_details() {
 		// phpcs:disable WordPress.Security.NonceVerification.Recommended
-		$is_resubscribe_request       = isset( $_REQUEST['resubscribe'] ) ? 'shop_subscription' === \get_post_type( absint( $_REQUEST['resubscribe'] ) ) : false;
+		$is_resubscribe_request       = isset( $_REQUEST['resubscribe'] ) ? true : false;
```

Comment:
Nit: We don't need the ternary, `isset` should return true or false.

Reply:
You're right, I was blindly following the convention set be the subsequent lines. Fixed in 3b1d8e2f99f9